src/main/java/core/basesyntax/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.bookstore.controller;
+
+import core.basesyntax.bookstore.dto.book.CreateBookRequestDto;
+import core.basesyntax.bookstore.service.BookService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/books")
+@RequiredArgsConstructor
+public class BookController {
+    private final BookService bookService;
+
+    @GetMapping
+    public ResponseEntity<?> getAllBooks() {
```

Comment:
you can return an object without ResponseEntity wrapping

Reply:
I'd like to keep it for aesthetics, if it's not crucial